draft-ietf-webtrans-http2.md

```diff
@@ -1171,6 +1171,50 @@ parameter (see {{h2-settings}}).  Clients can select the associated upgrade
 token, if applicable, to use when establishing a new session, ensuring that
 servers will always know the syntax in use for every incoming request.
 
+# Use Outside of HTTP/2
+
+## Other HTTP Versions
+
+This document specifies a protocol that provides WebTransport {{OVERVIEW}}
+functionality when transported using HTTP/2 {{HTTP2}}.  Because the
+mechanisms defined in this document rely only on generic HTTP semantics, they
+enable many WebTransport capabilities when using any transport that provides a
+bidirectional stream of bytes, including other HTTP versions.  However, this protocol MUST NOT be used within HTTP/3, as the protocol defined in {{WEBTRANSPORT-H3}} provides a higher fidelity mapping to the underlying transport, for example unreliable datagram capabilities.
```

Comment:
This text doesn't really motivate the MUST NOT.

I assume the real reason is that it just won't work, because WT-H3 uses the same Upgrade Token but probably won't respect most of these capsules.